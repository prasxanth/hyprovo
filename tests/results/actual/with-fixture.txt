2021-11-24 22:51:24.485 - INFO - Pass ✓... (test-add-variable): sum should be 2
2021-11-24 22:51:24.486 - ERROR - Fail ✗... (test-add-variable): sum should be 3
2021-11-24 22:51:24.491 - INFO - Pass ✓... (test-sub-variable): difference should be 0
2021-11-24 22:51:24.492 - ERROR - Fail ✗... (test-sub-variable): difference should be 4
