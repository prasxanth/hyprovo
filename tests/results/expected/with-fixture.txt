2021-11-24 22:50:31.296 - INFO - Pass ✓... (test-add-variable): sum should be 2
2021-11-24 22:50:31.298 - ERROR - Fail ✗... (test-add-variable): sum should be 3
2021-11-24 22:50:31.302 - INFO - Pass ✓... (test-sub-variable): difference should be 0
2021-11-24 22:50:31.303 - ERROR - Fail ✗... (test-sub-variable): difference should be 4
